### DISCUSSION TITLE
Customize the Way Payload Looks

### DISCUSSION THREAD

**Discussion by diyuksh** [payloadcms/payload#510] [Feature Requests & Ideas]
Hey,

I very recently ( Like Yesterday ) Started using Payload and I love it and am going to use it for production and client projects as well but I was wondering if there is a way we can customize the way the UI looks like changing the logo and things as such, since the old cms I used to use has a option for us to create a JS file to update colors, src for the logo and the dictionary for the whole UI, I feel like it could come in handy when delivering projects to high end clients and maybe even add external CSS, Scripts to make the UI more responsive.

Just a idea for enhancement would love to see it since I'm a beginner frontend developer as well would even love to fork the project for further enchantments.

Have a nice day/night everyone!

**Comment by DanRibbens** [comment]
Hey @outdatedx,

Thanks for the feedback! 

Turns out there is a lot you can do already for customizing the UI, CSS, logos, and more.

The docs talk about it in a few places:
- Customzing css - https://payloadcms.com/docs/admin/customizing-css
- Custom components - https://payloadcms.com/docs/admin/components

And this blog post explains exactly what to do to white label it for your clients: https://payloadcms.com/blog/white-label-admin-ui

I'm curious what you have in mind with "Scripts to make the UI more responsive", care to elaborate?

If there are changes to be made that everyone can benefit from, feel free to discuss it here and if you're up for contributing consider opening a pull request.

Cheers!

**Reply by diyuksh** [reply]
Oh I think I missed this in the docs 

Scripts for like animation libraries; it is a small change but would love to see it, and not only for Animation but could be for more.

**Reply by DanRibbens** [reply]
I think the best way to add scripts would be to make a custom index.html.

See `indexHTML` in https://payloadcms.com/docs/admin/overview#admin-options

Injecting your own scripts there is simple to do. Does that cover your use-case?
